### DISCUSSION TITLE
Multiple downloads by playlist & album

### DISCUSSION THREAD

**Discussion by chdlc** [KRTirtho/spotube#1117] [Ideas]
I think it'd be great to have an option available for downloading a playlist or album, then every time a song is added to a playlist the new song gets downloaded automatically.

**Comment by nepalivai** [comment]
There is a way to do that see issue #1069.
## note:- I don't know if it will download the new song every time you add new song.

**Reply by chdlc** [reply]
Oh my... I think I tried that yesterday and download action was still grey. Anyways, I didn't know about the YouTube API limitation so I'll be very careful.

I may need to validate if new songs are downloaded automatically. Thanks.
